src/main/fc/settings.yaml

```diff
@@ -382,15 +382,15 @@ groups:
     members:
       - name: min_throttle
         field: minthrottle
-        min: PWM_RANGE_ZERO
+        min: PWM_RANGE_MIN
         max: PWM_RANGE_MAX
       - name: max_throttle
         field: maxthrottle
-        min: PWM_RANGE_ZERO
+        min: PWM_RANGE_MIN
         max: PWM_RANGE_MAX
       - name: min_command
         field: mincommand
-        min: PWM_RANGE_ZERO
+        min: PWM_RANGE_MIN
```

Comment:
In some cases `min_command` has to be lower than 1000

Reply:
Should be good now. I put back min_command min to 0.